rclcpp/src/rclcpp/parameter_client.cpp

```diff
@@ -420,6 +472,42 @@ SyncParametersClient::set_parameters(
   return std::vector<rcl_interfaces::msg::SetParametersResult>();
 }
 
+std::vector<rcl_interfaces::msg::SetParametersResult>
+SyncParametersClient::delete_parameters(
+  const std::vector<std::string> & parameters_names,
+  std::chrono::nanoseconds timeout)
+{
+  auto f = async_parameters_client_->delete_parameters(parameters_names);
```

Comment:
This is just a suggestion, it is fine as-is, but if you had a method on the async parameter client that prepared the arguments for `set_parameters()` and returned them, you could get them here and call the sync parameter client's `set_parameters` rather than duplicating the logic in these two new methods.

Reply:
using a duplicate method seems to be the standard in the file.

Reply:
That's fine, just a suggestion.